### PR TITLE
Fix #15380 Advisor error

### LINKED ISSUE #15380: Status - Advisor error
5.0.0-alpha1

Select Status and then Advisor

Result:
Error in processing request 
Error code 500 
Error text Internal Server Error (rejected) 

Expected result:
Advisory report

PHP logs the following error (server name replaced with xxxx)
PHP Fatal error:  Uncaught TypeError: trim() expects parameter 1 to be string, int given in xxxxxxxxxxxxxxxxxx\\libraries\\classes\\SysInfoWINNT.php:88\nStack trace:\n#0 xxxxxxxxxxxxxxxx\\libraries\\classes\\SysInfoWINNT.php(88): trim(2048)\n#1 xxxxxxxxxxxxxxxxxxxxxxxxxx\\libraries\\classes\\SysInfoWINNT.php(116): PhpMyAdmin\\SysInfoWINNT->_getWMI('Win32_PageFileU...')\n#2 xxxxxxxxxxxxxxxxxxxxxxx\\libraries\\classes\\Advisor.php(229): PhpMyAdmin\\SysInfoWINNT->memory()\n#3 xxxxxxxxxxxxxxxxxxxxxxxxxx\\libraries\\classes\\Controllers\\Server\\Status\\AdvisorController.php(53): PhpMyAdmin\\Advisor->run()\n#4 xxxxxxxxxxxxxxxxxxx\\server_status_advisor.php(33): PhpMyAdmin\\Controllers\\Server\\Status\\AdvisorController->index()\n#5 {main}\n  thrown in xxxxxxxxxxxxxxxxxxxxxxxx\\libraries\\classes\\SysInfoWINNT.php on line 88

Server configuration
- Operating system: Windows 10 (1903) x64
- Web server: Apache 2.4.39 x64
- Database version: MySQL 8.0.16
- PHP version: 7.3.7
- phpMyAdmin version: 5.0.0-alpha1

### PR DESCRIPTION
**Pull request by Tithugues**
Fixes  #15380

### CODE REVIEW

**Comment by williamdes** [comment]
Can you please rebase onto `QA_4_9` since all bug fixes are for 4.9.x

**Comment by williamdes** [comment]
or QA_5_0?

**Comment by Tithugues** [comment]
@williamdes , the branch is based on QA_5_0. Is it OK?

**Review by MauricioFauth — APPROVED** [review]

**Comment by Tithugues** [comment]
Hi,
Is there anything that prevents this branch to be merged in QA_5_0 please?

**Comment by williamdes** [comment]
@Tithugues if you rebase onto `QA_4_9` I will merge it :)
Do not forget to change the PR target

**Comment by Tithugues** [comment]
Done!

File: libraries/classes/SysInfoWINNT.php [hunk 1]
```diff
@@ -9,6 +9,7 @@
 
 use COM;
 use PhpMyAdmin\SysInfoBase;
+use function is_string;
```
Comment (williamdes) [inline libraries/classes/SysInfoWINNT.php]: NOTE: php 5.6+
I will remove it when merging in `QA_4_9` and add it back when merging `QA_4_9` into `QA_5_0`

**Review by williamdes — APPROVED** [review]

**Review by williamdes — COMMENTED** [review]
Merged, thanks for the fix @Tithugues

File: libraries/classes/SysInfoWINNT.php (hunk 1 above)
Comment (Tithugues) [reply libraries/classes/SysInfoWINNT.php]: :+1:

**Pull request merged**